src/lib/operators/join_hash.cpp

```diff
@@ -382,9 +382,8 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
   number of hash tables that need to be looked into to just 1.
   */
   void _probe(const RadixContainer<RightType>& radix_container,
-              const std::vector<std::shared_ptr<HashTable<LeftType>>>& hashtables,
-              std::vector<std::shared_ptr<PosList>>& pos_list_left,
-              std::vector<std::shared_ptr<PosList>>& pos_list_right) {
+              const std::vector<std::shared_ptr<HashTable<LeftType>>>& hashtables, std::vector<PosList>& pos_list_left,
```

Comment:
`const`?

Reply:
At the probing stage the hashtables are all done and essentially "read-only" at this point. Does it hurt to pass it as `const` reference here?

Reply:
Nevermind. I didn't realize that `pos_list_left` was the output